browser/src/Editor/NeovimEditor/markdown.ts

```diff
@@ -4,8 +4,7 @@ import * as marked from "marked"
 import * as Log from "./../../Log"
 import { IGrammarPerLine, IGrammarToken } from "./../../Services/SyntaxHighlighting/TokenGenerator"
 
-// tslint:disable-next-line
-const { default: DOMPurify } = require("dompurify")
+import * as DOMPurify from "dompurify"
```

Comment:
I defeated you little bug:stuck_out_tongue:

Reply:
Yeah I had some weird behaviour with that module (hence the bizarre import strategy) which the new changes to webpack seem to have normalised